docker-compose.yml

```diff
@@ -119,12 +121,9 @@ services:
       - COCKROACH_PASSWORD=${DB_PASSWORD:-very_unsafe_password_replace_me}
     volumes:
       - data:/var/lib/cockroach
-    expose:
-      - "8080"
-      - "26257"
     ports:
-      - "26257:26257"
-      - "8080:8080"
+      - "26257"
+      - "8080"
```

Comment:
This doesn't stop the ports from being exposed to the host. Not specifying the host port here means that Docker will assign it to a random port on the host. People can still do a port scan and find the open ports.

We should:
1. For ports that are used by other containers but not the host: move to `expose` section. E.g., postgres `26257`
2. For ports that are used by other containers and the host: move to `ports` section, E.g., cardinal `4040`
3. For ports that aren't used by other containers and the host: remove entirely. E.g., cockroach console `8080`

Reply:
updated